Apps/Src/UpdateVelocity.c

```diff
@@ -17,9 +17,14 @@
 #define NEUTRAL_PEDALS_PERCENT 25
 #define REGEN_RANGE (NEUTRAL_PEDALS_PERCENT - UNTOUCH_PEDALS_PERCENT)
 
+static float cruiseSpeed = 0; //cruising speed
+static State cruiseSet = OFF; //whether cruise is set
+static State cruiseEnable = OFF; //whether cruise is enabled
+static State prevEnable = OFF; //extra state variable for edge detector
+static State prevSet = OFF; //extra state variable for edge detector
```

Comment:
I've used it since our Switches Driver is built with the State return type, and I'd rather just use it here instead of casting it to `bool` for readability purposes.

Reply:
> I'm not a huge fan of the `enum State` to be honest. I feel like `bool` does the same thing.

That's what I said. It's stupid.